poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.5 and should not be changed by hand.
```

Comment:
Please use poetry 1.6.1 for this, sometimes new versions introduce changes are not compatible